src/main/resources/application.conf

```diff
@@ -28,6 +29,17 @@ mantis {
   # secure-random-algo = "NativePRNG"
 
   network {
+
+    # riemann {
```

Comment:
I'm not sure if `network` is the best section to put it. So far it was only intended for eth network stuff. Maybe we can put riemann config in separate section (like we did with `metrics`)?

Reply:
moved